env/testtoolshed/group_vars/toolshedservers/vars.yml

```diff
@@ -17,29 +17,23 @@ toolshed_dbname: galaxy_testtoolshed
 ## used by: other vars and various templates
 galaxy_toolshed_instance_codename: test
 galaxy_toolshed_instance_hostname: testtoolshed.g2.bx.psu.edu
-
+galaxy_toolshed_gunicorn_workers: 2
+galaxy_toolshed_gunicorn_bind: "127.0.0.1:9009"
+galaxy_toolshed_gunicorn_max_requests: 50000
+galaxy_toolshed_gunicorn_max_requests_jitter: 5000
```

Comment:
We should probably decrease that if we're running standalone hgweb. That's a good way to end up with entries in the database but not within the hgweb config.